src/os/Makefile.am

```diff
@@ -46,6 +46,8 @@ libos_a_SOURCES += \
 	os/bluestore/ExtentFreelistManager.cc \
 	os/bluestore/FreelistManager.cc \
 	os/bluestore/KernelDevice.cc \
+	os/bluestore/BitMapAllocator.cc \
+	os/bluestore/BitAllocator.cc \
 	os/bluestore/StupidAllocator.cc
```

Comment:
Please add the header files to the headers list in the same file.

There is also a cmake file that should be updated.